deploy/crds/apps.3scale.net_apimanagerbackups_crd.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: apimanagerbackups.apps.3scale.net
+spec:
+  group: apps.3scale.net
+  names:
+    kind: APIManagerBackup
+    listKind: APIManagerBackupList
+    plural: apimanagerbackups
+    singular: apimanagerbackup
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: APIManagerBackup is the Schema for the apimanagerbackups API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: APIManagerBackupSpec defines the desired state of APIManagerBackup
+          properties:
+            apiManagerName:
+              description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+                Important: Run "operator-sdk generate k8s" to regenerate code after
+                modifying this file Add custom validation using kubebuilder tags:
+                https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html'
+              minLength: 1
+              type: string
+            backupSource:
+              properties:
+                persistentVolumeClaim:
+                  description: 'Ways to define a PVC creation: Define VolumeName OR
+                    Define Resources. When VolumeName is specified resources is not
+                    needed: Detailed info: https://docs.okd.io/3.11/dev_guide/persistent_volumes.html#persistent-volumes-volumes-and-claim-prebinding'
+                  properties:
+                    resources:
+                      properties:
+                        requests:
+                          type: string
+                      required:
+                      - requests
+                      type: object
+                    storageClass:
```

Comment:
storageClass inside `PersistentVolumeClaimResourcesSpec`??

Reply:
It is not inside `PersistentVolumeClaimResourcesSpec`. It is inside `PersistentVolumeClaimBackupSource`

Reply:
I know, it was not a question, but a proposal.

The options are providing either:
* volumeName
* Resources and (optionally) storageClass

So those three attributes of `PersistentVolumeClaimBackupSource`  are not sibling attributes. There are two groups. Hence my proposal.

Reply:
This section intended to mimic the Kubernetes PVC settings, where they are in the same level indeed https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#persistentvolumeclaimspec-v1-core.
I personally prefer to keep it as similar as PVC itself.

Reply:
ok, being consistent with PVC structure is desired :+1: